Source: Gaute Hope <eg@gaute.vetsj.com>, Cecil Curry <leycec@gmail.com>
Upstream: no
Reason: Install stuff in a standard location; avoid stripping binaries.
diff -ru old/Makefile new/Makefile
--- old/Makefile	2010-03-06 19:38:28.540091351 +1300
+++ new/Makefile	2010-03-06 19:44:45.109218475 +1300
@@ -9,7 +9,7 @@
                RM = rm -f
              TAGS = ctags
            MFLAGS = -$(MAKEFLAGS)
-     INSTPGMFLAGS = -c -s
+     INSTPGMFLAGS = -c
      INSTMANFLAGS = -c
      TOP_INCLUDES = -I$(INCROOT)
       CDEBUGFLAGS = -O
@@ -21,9 +21,9 @@
            RM_CMD = $(RM) *.CKP *.ln *.BAK *.bak *.o core errs ,* *~ *.a .emacs_* tags TAGS make.log MakeOut
          IRULESRC = $(CONFIGDIR)
         IMAKE_CMD = $(IMAKE) -DUseInstalled -I$(IRULESRC) $(IMAKE_DEFINES)
-           BINDIR = $(DESTDIR)/usr/bin/X11
+           BINDIR = $(DESTDIR)/usr/bin
           INCROOT = $(DESTDIR)/usr/include
-          MANPATH = $(DESTDIR)/usr/catman/x11_man
+          MANPATH = $(DESTDIR)/usr/share/man
     MANSOURCEPATH = $(MANPATH)/man
            MANDIR = $(MANSOURCEPATH)1
             IMAKE = imake
